packages/sources/instruxi-sxt/test-payload.json

```diff
@@ -0,0 +1,12 @@
+{
+    "data": {
+    "endpoint": "total_reserve",
+      "BISCUIT_ATTESTATIONS": "example_biscuit_attestations_token",
+      "BISCUIT_BLOCKCHAINS": "example_biscuit_blockchains_token",
+      "CHAIN_ID": "chain_id",
+      "ASSET_CONTRACT_ADDRESS": "example_address",
+      "TOKEN_CONTRACT_ADDRESS": "example_address",
+      "NAMESPACE": "example_namespace"
```

Comment:
this is used for soak testing, you'll need to put real values here. Are you allowed to do so?

Reply:
I added real values that return a proper response